wire-ios/Wire-iOS Tests/ImageCache/DecodeImageOperationTests.swift

```diff
@@ -16,27 +16,39 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-@testable import Wire
+import WireUITesting
 import XCTest
 
+@testable import Wire
+
 final class DecodeImageOperationTests: XCTestCase {
 
-    var operationQueue: OperationQueue!
-    var sut: UIImageView!
+    // MARK: - Properties
+
+    private var snapshotHelper: SnapshotHelper!
+    private var operationQueue: OperationQueue!
+    private var sut: UIImageView!
+
+    // MARK: - setUp
 
     override func setUp() {
         super.setUp()
+        snapshotHelper = SnapshotHelper()
         operationQueue = OperationQueue()
         sut = UIImageView(frame: CGRect(x: 0, y: 0, width: 300, height: 300))
         sut.contentMode = .scaleAspectFit
     }
 
+    // MARK: - tearDown
+
```

Comment:
I think you forgot to nil `snapshotHelper` this time

Reply:
Good catch! Done: https://github.com/wireapp/wire-ios/commit/110626e3f4b056af045b6bc37c80c371243187c8